src/libponyrt/mem/pool.h

```diff
@@ -43,6 +49,28 @@ size_t ponyint_pool_used_size(size_t index);
 
 size_t ponyint_pool_adjust_size(size_t size);
 
+#ifndef POOL_USE_DEFAULT
```

Comment:
I'd like to see the order reversed here.

Can you make this say `#ifdef POOL_USE_DEFAULT` and make the default case come first?

Reply:
switched